backends/advanced/src/advanced_omi_backend/chat_service.py

```
@@ -312,11 +314,26 @@ async def format_conversation_context(
+        if include_obsidian_memory:
+            try:
+                obsidian_service = get_obsidian_service()
+                obsidian_context = obsidian_service.search_obsidian(current_message)
+                if obsidian_context:
+                    context_parts.append("# Relevant Obsidian Notes:")
+                    for entry in obsidian_context:
+                        context_parts.append(entry)
+                    context_parts.append("")
+                    logger.info(f"Added {len(obsidian_context)} Obsidian notes to context")
+            except Exception as e:
+                logger.error(f"Failed to get Obsidian context: {e}")
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)